source/guides/maintenance/container-image-modify.rst

```diff
@@ -131,21 +133,27 @@ OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build c
       Version URL:       https://cdn.download.clearlinux.org/update
       Content URL:       https://cdn.download.clearlinux.org/update
 
-Example 3: Change Clear Linux OS version (multi-stage Dockerfiles)
-===================================================================
 
-This example shows how to rebuild multi-stage Dockerfiles to use a specific
-OS version.
+Example 3: Change |CL| version (multi-stage build)
+**************************************************
+
+This example shows how to rebuild the cgit Dockerfile to use a specific |CL|
+version. The clearlinux/cgit Dockerfile has a multi-stage build with multiple
+layers: *os-core*, *httpd*, and *cgit*. This can be used as reference for
+building other multi-stage images with any number of layers. 
+
 
 .. important::
 
-   All upper layers of multi-stage Dockerfiles inherit the
-   |CL| version from the base layer. Rebuild the all underlying base layers against the desired OS version. In this example, four base layers must be rebuilt.
+   All upper layers of multi-stage Dockerfiles inherit the |CL| version from
+   the base layer. Rebuild the all underlying base layers against the desired
+   OS version. In this example, four base layers must be rebuilt.
 
-#. Change directory into :file:`dockerfiles`.
 
-First layer
------------
+First layer: os-core
+--------------------
+
+#. Change directory into :file:`dockerfiles`.
```

Comment:
Ln 166:  When I run this command, output shows `update successful`; however, an error message appears just like above: 
`swupd: error while loading shared libraries: libsystemd.so.0: cannot open shared object file: No such file or directory`

Reply:
This indicates another issue with your environment. No libsystemd tells me you're likely in a console inside a container when you shouldn't be.